Commands/Modules/Converter.cs

```diff
@@ -0,0 +1,152 @@
+    { ".gif", async (input, output, fps) => await ConvertVideoToGif(input, output, fps) },
+    { ".wmv", async (input, output, fps) => await ConvertVideo(input, output) },
+    { ".flv", async (input, output, fps) => await ConvertVideo(input, output) },
+    { ".mpg", async (input, output, fps) => await ConvertVideo(input, output) },
+    
+    // Image formats
+    { ".png", async (input, output, fps) => await ConvertImage(input, output) },
+    { ".jpg", async (input, output, fps) => await ConvertImage(input, output) },
+    { ".webp", async (input, output, fps) => await ConvertImage(input, output) },
+    { ".bmp", async (input, output, fps) => await ConvertImage(input, output) },
+    { ".tiff", async (input, output, fps) => await ConvertImage(input, output) },
+    { ".pbm", async (input, output, fps) => await ConvertImage(input, output) },
+    { ".tga", async (input, output, fps) => await ConvertImage(input, output) },
+    { ".qoi", async (input, output, fps) => await ConvertImage(input, output) }
+    };
+
+    [SlashCommand("convert", "Convert a file format to another one")]
+    public async Task ConvertAsync(
+        [Summary(description: "The video or image to convert")] Discord.Attachment attachment,
+        [Choice("gif", ".gif"), Choice("mp4", ".mp4"), Choice("mpg", ".mpg"), Choice("avi", ".avi"), Choice("mov", ".mov"), Choice("mkv", ".mkv"), Choice("png", ".png"), Choice("jpg", ".jpg"), Choice("webp", ".webp"), Choice("bmp", ".bmp"), Choice("tiff", ".tiff"), Choice("wmv", ".wmv"), Choice("flv", ".flv"), Choice("pbm", ".pbm"), Choice("tga", ".tga"), Choice("qoi", ".qoi")] string outputFormat,
+        [Summary("fps", description: "If using Gif, what fps should it have")] int fps = 30,
+        [Summary(description: "A message to send with the converted file")] string message = "",
+        [Summary(description: "If the file should be sent as a temporary message, that's only visible to you")] bool ephemeral = false)
+    {
+        string inputFilePath = Path.GetTempFileName();
+        string outputFilePath = Path.Combine(Path.GetTempPath(), inputFilePath + outputFormat);
+
+        // Acknowledge the command
+        await DeferAsync(ephemeral);
+        await DownloadFileAsync(attachment.Url, inputFilePath);
+
+        try
+        {
+            if (conversionMap.TryGetValue(outputFormat, out var converter))
+            {
+                if (Path.GetExtension(attachment.Filename) == outputFormat)
+                {
+                    await FollowupAsync($"Silly, you are converting from {Path.GetExtension(attachment.Filename)} to {outputFormat}.", ephemeral: ephemeral);
+                    return;
+                }
+                await converter(inputFilePath, outputFilePath, fps);
+            }
+            else
+            {
+                await FollowupAsync("Unsupported conversion type.", ephemeral: ephemeral);
+                return;
+            }
```

Comment:
if could be inverted to reduce nesting
Not sure if this is more readable tho

```suggestion
            if (!conversionMap.TryGetValue(outputFormat, out var converter))
            {
                await FollowupAsync("Unsupported conversion type.", ephemeral: ephemeral);
                return;
            }

            if (Path.GetExtension(attachment.Filename) == outputFormat)
            {
                await FollowupAsync($"Silly, you are converting from {Path.GetExtension(attachment.Filename)} to {outputFormat}.", ephemeral: ephemeral);
                return;
            }
            await converter(inputFilePath, outputFilePath, fps);
```